src/test/java/guitests/ContextMenuTests.java

```diff
@@ -81,6 +80,16 @@ public void test2() {
     }
 
     /**
+     * Tests selecting "Mark all below as read" and "Mark all below as unread" context menu items
+     */
+    @Test
+    public void markAllBelowAsReadUnread_tenIssuesInListView_markIssue9andBelowReadUnread() {
+        bringSelectedAndIssuesBelowInFocus(7);
+        markIssuesBelow(7, true);
+        markIssuesBelow(7, false);
```

Comment:
I have added tests for checking whether `IssueSelectedEvent` is called upon a left click and a key press and not called upon a right click. I am also checking whether all context menu items are disabled for the empty list case so corner case is being checked. `Mark all below as read` and `Mark all below as unread` are also being checked thoroughly now so I think we are good now! :+1: